group_vars/maintenance.yml

```diff
@@ -19,6 +19,10 @@ galaxy_user:
   uid: 999
   shell: /bin/bash
 
+galaxy_cleanup_objectstores:
+  - objectstore_id: 's3_scratch_netapp01'
```

Comment:
We should add a scary warning here. Or any other safe-guard.

If we add the wrong object store here, we lose data. Essentially, we should only ever add here storages with the name `scratch` in its name.